tests/README.md

```diff
@@ -37,3 +37,49 @@ MOONBEAM_LOG="warn,rpc=trace" npm run test
 
 The test script will find available ports above 20000 in order to ensure that it doesn't conflict
 with any other running services.
+
+# Debugging a Moonbeam node
+
+The repository contains a pre-configured debugger configuration for VSCode with the **CodeLLDB**
+(`vadimcn.vscode-lldb`) extension. In the Debug tab of VSCode the dropdown at the top list 2
+configurations :
+
+- **Build & Launch Moonbeam Node (Linux)** : cargo build the node with debug profile and launch it.
+- **Launch Moonbeam Node (Linux)** : directly launch the node without calling cargo build. Usefull
```

Comment:
*useful

Also please explain precisely how to launch the node

Reply:
I added the explanation below this part.